xchange-ftx/src/main/java/org/knowm/xchange/ftx/service/FtxLendingService.java

```diff
@@ -0,0 +1,277 @@
+package org.knowm.xchange.ftx.service;
+
+import org.apache.commons.lang3.StringUtils;
+import org.knowm.xchange.ftx.FtxExchange;
+import org.knowm.xchange.ftx.dto.account.*;
+import org.knowm.xchange.utils.BigDecimalUtils;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class FtxLendingService extends FtxBaseService {
+
+  public FtxLendingService(FtxExchange exchange) {
+    super(exchange);
+  }
+
+  public FtxLendDataDto stopLending(String subaccount, String coin) {
+    return lend(subaccount, coin, 0, 0);
+  }
+
+  public List<FtxLendDataDto> stopLending(String subaccount, List<String> coins) {
+    return coins.stream()
+        .map(coin -> stopLending(subaccount, coin))
+        .collect(Collectors.toList());
+  }
+
+  public List<FtxLendDataDto> stopLending(String subaccount, String... coins) {
+    return stopLending(subaccount, Arrays.asList(coins));
+  }
+
+  public FtxLendDataDto lendAll(String subaccount, String coin, double rate) {
+    Objects.requireNonNull(coin);
+    if (StringUtils.isNotBlank(coin) || StringUtils.isNotEmpty(coin))
```

Comment:
redundant check here, isNotBlank is superset of isNotEmpty

Reply:
I change this

Reply:
dont need objects check either